docker/importer/importer.py

```diff
@@ -210,8 +210,10 @@ def _process_updates_git(self, source_repo):
     changed_entries = set()
     deleted_entries = set()
 
-    if source_repo.last_synced_hash:
-      # Syncing from a previous commit.
+    def _sync_from_previous_commit(repo):
```

Comment:
Can we pull this out into a top level method? I **think** we just need to pass `source_repo` here as an extra arg?

Reply:
Done in bfc7965